processor/stream/benchmark_test.go

```diff
@@ -48,7 +49,7 @@ func BenchmarkStreamProcessor(b *testing.B) {
 	}
 	//ensure to not hit rate limit as blocking wait would be measured otherwise
 	rl := rate.NewLimiter(rate.Limit(math.MaxFloat64-1), math.MaxInt32)
-	sp := &Processor{MaxEventSize: 300 * 1024}
+	sp := &Processor{MaxEventSize: 300 * 1024, metadataSchema: metadata.ModelSchema()}
```

Comment:
I think this should use one of the provided processors (eg. `stream.BackendProcessor`) instead of creating a new one.

Reply:
I've created two separate benchmarks, one for each of BackendProcessor and RUMV3Processor. They're slower now because they have modelDecoders, whereas the old benchmark had an empty set.